src/payment/coupon/data/service.js

```diff
@@ -1,6 +1,7 @@
 import pick from 'lodash.pick';
 
-import { handleRequestError, applyConfiguration } from '../../../common/serviceUtils';
+import { applyConfiguration } from '../../../common/serviceUtils';
```

Comment:
Is this import of `applyConfiguration` used?

Reply:
Yes it's in there.